src/common/com/intellij/plugins/haxe/lang/psi/impl/AnonymousHaxeTypeImpl.java

```diff
@@ -39,9 +36,12 @@ public AnonymousHaxeTypeImpl(@NotNull ASTNode node) {
   @NotNull
   @Override
   public List<HaxeType> getHaxeExtendsList() {
```

Comment:
A question:  Did you happen to check if the callers to getHaxeExtendsList() actually expect and process a list, or whether they expect the list to only contain one entry?